src/main/java/mate/academy/spring/controller/CinemaHallController.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import mate.academy.spring.dto.cinema.hall.CinemaHallRequestDto;
+import mate.academy.spring.dto.cinema.hall.CinemaHallResponseDto;
+import mate.academy.spring.mapper.CinemaHallDtoMapper;
+import mate.academy.spring.model.CinemaHall;
+import mate.academy.spring.service.CinemaHallService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema_halls")
```

Comment:
We shouldn't use underscores. Better to change it to:
```suggestion
@RequestMapping("/cinema-halls")
```
![image](https://github.com/mate-academy/jv-spring-rest/assets/96497090/6ac9a35e-19a3-484f-9ca1-248c712d68c8)